common/app/model/Asset.scala

```diff
@@ -82,20 +82,23 @@ object VideoAsset {
     VideoAsset(
       fields = Helpers.assetFieldsToMap(asset),
       mimeType = asset.mimeType,
-      url = asset.file )
+      url = asset.file,
+      secureUrl = asset.typeData.get.secureFile )
```

Comment:
@philwills Spooky coincidence! That must have caused a race condition down at Github. "two users, same message, same time = DDOS. Shutdown github immediately"